code/edge/src/main/java/com/adobe/marketing/mobile/EdgeJson.java

```diff
@@ -42,7 +42,6 @@ static class Xdm {
 
 			static final String EVENT_ID = "_id";
 			static final String EVENT_MERGE_ID = "eventMergeId";
-			static final String EVENT_TYPE = "eventType";
```

Comment:
Can you also:
- Remove BEACON and PLACE_CONTEXT as they are not being used?
- Move EVENT_MERGE_ID and COMMERCE to TestExperienceEventSchema as they are only used in that test file?